core/src/migrations/000065-lengthen-id-columns.ts

```diff
@@ -0,0 +1,65 @@
+import { config } from "actionhero";
+
+const tables = {
+  apiKeys: ["apiKey"],
+  apps: [],
+  destinationGroupMemberships: ["destinationId", "groupId"],
+  destinations: ["appId", "groupId"],
+  eventData: ["eventId"],
+  events: ["producerId", "profileId"],
+  exports: ["profileId", "destinationId"],
+  files: [],
+  groupMembers: ["profileId", "groupId"],
+  groupRules: ["groupId", "propertyId"],
+  groups: [],
+  imports: ["creatorId", "profileId"],
+  logs: ["ownerId"],
+  mappings: ["ownerId", "propertyId"],
+  notifications: [],
+  options: ["ownerId"],
+  permissions: ["ownerId"],
+  profileProperties: ["profileId", "propertyId"],
+  profiles: [],
+  properties: ["sourceId"],
+  propertyFilters: ["propertyId"],
+  runs: ["creatorId", "destinationId"],
+  schedules: ["sourceId"],
+  sessions: [],
+  settings: [],
+  setupSteps: [],
+  sources: ["appId"],
+  teamMembers: ["teamId"],
+  teams: [],
+};
+
+const runMigration = async ({ maxIdLength, migration, DataTypes }) => {
+  const changeColumn = async (tableName, columnName) => {
+    if (config.sequelize.dialect === "postgres") {
+      const query = `ALTER TABLE "${tableName}" ALTER COLUMN "${columnName}" SET DATA TYPE varchar(${maxIdLength}); `;
+      await migration.sequelize.query(query);
+    } else {
+      await migration.changeColumn(tableName, columnName, {
+        type: DataTypes.STRING(191),
+      });
+    }
```

Comment:
Super interesting (and another SQLite quirk) is that SQLite does not support altering a column. Its `ALTER TABLE` functions [are super limited](https://sqlite.org/lang_altertable.html).

But that's where Sequelize comes in to save the day. So ... this ends up still looking a little goofy, but at least we don't have to mess with constraints.